app.js

```diff
@@ -0,0 +1,21 @@
+const taskManager = () => {
+  const inputTodo = document.getElementById("inputTodo");
+  const addTodoButton = document.getElementById("addTodoButton");
+  const listTodo = document.getElementById("listTodo");
+  const addTodo = () => {
+    const inputText = inputTodo.value.trim();
+    if (inputText == "") {
+      alert("Enter Something!");
```

Comment:
Don't use DOM blocking calls like `alert`. This is a bad UX practice. Instead, we will figure out a way to show errors to users.
```suggestion
```

Reply:
alert is removed